poly/src/polynomial/univariate/dense.rs

```diff
@@ -506,13 +506,16 @@ mod tests {
         for degree in 0..70 {
             let poly = DensePolynomial::<Fr>::rand(degree, &mut rng);
             let neg = -poly.clone();
-            assert!((poly + neg).is_zero());
+            let result = poly + neg;
+            assert!(result.is_zero());
+            assert_eq!(result.degree(), 0);
```

Comment:
drive by improvement to the additive identity tests. It would have caught the bug fixed by #119